src/inverseRobot.js

```diff
@@ -24,7 +24,20 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const inversedRobot = {};
+  const robotKeys = Object.values(robot).reverse();
+  const robotValues = Object.keys(robot).reverse();
+
+  for (let i = 0; i < robotValues.length; i++) {
+    // eslint-disable-next-line max-len
+    if (robotKeys.indexOf(robotKeys[i]) !== robotKeys.lastIndexOf(robotKeys[i])) {
```

Comment:
Don't use arrays to store keys/values. Just refer to original robot using `for ... in` loop.